server/routes.js

```diff
@@ -121,6 +121,7 @@ module.exports = (app) => {
    * Groups.
    */
   app.post('/groups', aN.authenticateUserByJwt(), required('group'), groups.create); // Create a group. Option `role` to assign the caller directly (default to null).
+  app.post('/groups/create_from_github_repo', aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, required('payload'), connectedAccounts.createGroupFromGithubRepo)
```

Comment:
- resources should be nouns, not verbs; and the resource that is being created is really a group, not a group called "create_from_github_repo"
- use dash instead of underscore in URLs
- missing semicolon
- why isn't the JWT expiry checked? AFAIK only place where that should happen is `POST /subscriptions/refresh_token`
- why is this `POST /groups...` handled by `connectedAccounts` controller whereas line above is handled by `groups` controller?

Maybe have a single endpoint `POST /groups?flow=manual|github`? If too difficult to combine in a single route (because security middleware differs), maybe do something like:

```
app.post('/groups', ifParam('flow', 'manual'), aN.authenticateUserByJwt(), required('group'), groups.create);
app.post('/groups', ifParam('flow', 'github'), aN.authenticateAppByApiKey, aN.parseJwtNoExpiryCheck, required('payload'), groups.createFromGithub);

function ifParam(param, value) {
  return (req, res, next) => {
    if (!req.params[param]) return next(new Error(`missing parameter ${param}`));
    if (req.params[param] !== value) return next('route');
    next();
  }
}
```